verification/curator-service/test/sources.test.ts

```diff
@@ -1,6 +1,27 @@
 import request from 'supertest';
 import app from '../src/index';
 
+import mongoose from 'mongoose';
+
+beforeAll(async () => {
+    await mongoose.connect(
+        // This is provided by jest-mongodb.
+        // The `else testurl` is to appease Typescript.
+        process.env.MONGO_URL || 'testurl',
```

Comment:
we could optionally use an assertion here (and in index.ts), but that might be too heavy-handed here

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions

Reply:
I'm planning to plug in the env validator that's already plugged into the data validator service, this will ensure the env is valid and this test will be moot, I hope it's okay if I just leave it as is for now.

Reply:
yep sg!